unfoldingWord/translationCore
# Feature #3186/migrate alignment data

#### This pull request addresses:

Migrates alignment data to tc version 2.  Changes 'strongs' to 'strong'.  Fixes mistakes in target language occurrence(s) counts.


#### How to test this pull request:

When you open a tc version 1 project, the manifest will get updated to tc version 2. Also alignment data (in '.apps/translationCore/alignmentData') should have references to strongs converted to strong, as well as occurrence/occurrences cleanup.



---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3749)
